cloud/pkg/devicecontroller/apis/devices/v1alpha1/device_instance_types.go

```diff
@@ -139,7 +139,7 @@ type Twin struct {
 // TwinProperty represents the device property for which an Expected/Actual state can be defined.
 type TwinProperty struct {
 	// Required: The value for this property.
-	Value string `json:"value,omitempty"`
+	Value string `json:"value,"`
```

Comment:
@subpathdev does this change warrant a change in https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/devicecontroller/apis/devices/v1alpha1/zz_generated.deepcopy.go
Could you please run this script https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/devicecontroller/apis/devices/v1alpha1/zz_generated.deepcopy.go and verify if the deepcopy file is changed ?

Reply:
I didn't change https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/devicecontroller/apis/devices/v1alpha1/zz_generated.deepcopy.go. Why we have to change this file? I have a running example with the changes which are provided by this pr. (Only this changes)

I have also executed the update-codegen.sh script and nothing has changed.

Reply:
Usually when we modify the structures in either the device_instance_types or device_model_types, it requires changes in the deepcopy file which is a generated file.

Reply:
If the script has not changed your deepcopy file I guess there is no change required in the deepcopy file for this change